pyproject.toml

```diff
@@ -57,7 +57,7 @@ pytest-cov = {version = ">=5.0.0", optional = true}
 datasets = ">=2.19.0"
 imagecodecs = { version = ">=2024.1.1", optional = true }
 pyav = ">=12.0.5"
-rerun-sdk = ">=0.15.1"
+rerun-sdk = ">=0.19.0"
```

Comment:
We're using poetry to manage dependencies: for the tests to pass, the `poetry.lock` file needs to be in sync with what's in the `pyproject.toml`. Once you have poetry setup (install it in a different virtual env than the one you're using lerobot with), you can simply do this with `poetry lock`.